src/components/nav/style.css

```
@@ -8,6 +8,24 @@
   z-index: 30;
 }
 
+@media only screen and (min-width: 320px) {
+  .search {
+    width: 300px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .search {
+    width: 100%;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  .search {
+    width: 100%;
+  }
+}
```

```suggestion
@media only screen and (min-width: 768px) {
  .search {
    width: 100%;
  }
}
```

1 is enough for this case.